cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -1341,8 +1341,16 @@ queryResult eAcq = pure eAcq
 
 toEpochInfo :: EraHistory CardanoMode -> EpochInfo (Either Text)
 toEpochInfo (EraHistory _ interpreter) =
-  hoistEpochInfo (first (Text.pack . show ) . runExcept)
-    $ Consensus.interpreterToEpochInfo interpreter
+  hoistEpochInfo (first (Text.pack . show) . runExcept)
+    $ Consensus.interpreterToEpochInfo (Consensus.unsafeExtendSafeZone interpreter)
+
+newtype Unsafe a = Unsafe { unsafe :: a } deriving (Eq, Show)
+
+unsafeToEpochInfo :: EraHistory CardanoMode -> Unsafe (EpochInfo (Either Text))
```

Comment:
Similarly, I'd suggest naming this `toEpochAssumingNoUpcomingHardFork` or something like that.

Reply:
`toBeyondHorizonEpochInfo`?

Reply:
This is now `toTentativeEpochInfo`.